common/src/main/java/com/microsoft/identity/common/internal/cache/ADALTokenCacheItem.java

```diff
@@ -83,8 +83,9 @@
     ADALTokenCacheItem(final AzureActiveDirectoryOAuth2Strategy strategy,
                        final AzureActiveDirectoryAuthorizationRequest request,
                        final AzureActiveDirectoryTokenResponse response) {
-        AzureActiveDirectoryAccount account = strategy.createAccount(response);
         String issuerCacheIdentifier = strategy.getIssuerCacheIdentifier(request);
+        AzureActiveDirectoryAccount account = strategy.createAccount(response);
```

Comment:
Future opportunity: To have createAccount actually deal with the issuer cache identifier.